modules/telco-ran-measured-kpi-results.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-du-overview.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-measured-kpi-results_{context}"]
+= KPI test results overview
+
+All results are taken from 12-hour test runs.
+
+Realtime kernel KPI test::
+Workload nodes running the realtime kernel are validated to these performance KPIs:
+
+* No `oslat` samples greater than 20µs
```

Comment:
If one item in the list is a complete sentence, then all items in the list must be completes sentences per ISG
```suggestion
* No `oslat` samples greater than 20µs.
```